src/Navbar.js

```diff
@@ -83,6 +84,7 @@ class Navbar extends Component {
 Navbar.propTypes = {
   brand: PropTypes.node,
   children: PropTypes.node,
+  mobileNav: PropTypes.node,
```

Comment:
Cool, let's add some description here for documentation (even thought the name is very clear) :)